lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/fs.rb

```diff
@@ -490,6 +490,7 @@ def cmd_download(*args)
     # Go through each source item and download them
     src_items.each { |src|
       src = ::File.expand_path(src, client.fs.dir.getwd)
+      dest = ::File.expand_path(dest)
```

Comment:
Unfortunately this only expands the path according to your local file system. This doesn't do it in the context of the remote file system. And the function to do this is broken on Meterpreter as I listed at https://github.com/rapid7/metasploit-framework/issues/14357

Reply:
Nm this is the local path or file we are downloading the remote file to. I got confused here. This is fine.